File: JuvenilesColon.API/Models/Jugador.cs
using System;
using System.Collections.Generic;

namespace JuvenilesColon.API.Models
{
    public class Jugador
    {
        public int Id { get; set; }
        public string Apellido { get; set; }
        public string Nombre { get; set; }
        public DateTime? FechaNacimiento { get; set; }
        public DateTime? FechaIngreso { get; set; }
        public double? Altura { get; set; }
        public ICollection<Foto> Fotos { get; set; }
        public ICollection<Posicion> Posiciones { get; set; }

        //Ojo, esto deberia ser con FK
        public string Localidad { get; set; }
    }
}

File: JuvenilesColon-SPA/src/app/_models/jugador.ts
import { Foto } from './foto';

export interface Jugador {
    id: number;
    apellido: string;
    nombre: string;
    fechaNacimiento: Date;
    fechaIngreso: Date;
    altura: number;
    fotos: Foto[];
    // public ICollection<Posicion> Posiciones { get; set; }
    localidad: string;
}